msu/utilities/exceptions.nut

```diff
@@ -0,0 +1,9 @@
+::MSU.Exception <- {
+	KeyNotFound = "Key not found",
```

Comment:
As alluded to in a few comments I have made I think several of these exceptions should be functions that take in a string arg and print that in the log. For example `KeyNotFound = function( _key ) return "Key not found: " + _key;`